modules/dag-shared/src/main/scala/org/tessellation/dag/snapshot/GlobalSnapshotTips.scala

```diff
@@ -0,0 +1,10 @@
+package org.tessellation.dag.snapshot
+
+import derevo.cats.{eqv, show}
+import derevo.derive
+
+@derive(eqv, show)
+case class GlobalSnapshotTips(
```

Comment:
One thing to mention, it's not necessary to repeat `tips` again, so instead of:
```scala
case class GlobalSnapshotTips(
   deprecatedTips: Set[DeprecatedTip],
   activeTips: Set[ActiveTip]
 )
 ```
 I'd see it as:
 ```scala
 case class GlobalSnapshotTips(
   deprecated: Set[DeprecatedTip],
   active: Set[ActiveTip]
 )
 ```
 
 Second thing is that if `active` are the tips from the previous snapshots, maybe it would be easier to understand by changing the name `active` into `previous`, `indirect`, `extended` or `prolonged`?

Reply:
I agree with removing the `tips` suffix, it's unnecessary.

Regarding the naming. Myabe `remainedActive`?

Reply:
Or `stillActive`